tests/test_all_rules.py

```diff
@@ -81,6 +81,10 @@ def test_duplicate_file_names(self):
         if duplicates:
             self.fail(f"Found duplicated file names {duplicates}")
 
+    def test_rule_type_changes(self):
+        """Test that a rule type did not change for a locked version"""
+        default_version_lock.manage_versions(self.production_rules)
```

Comment:
We should probably assert that no rules fail and/or add a unittest to assert that this will fail when changing types.

Reply:
I don't think we need to - I manually tested and that code won't change regularly